projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/TestJobDeployTempCredsIT.java

```diff
@@ -98,6 +104,9 @@ public void setup() throws Exception {
                                 String.format(
                                     "/data-jobs/for-team/%s/jobs/%s",
                                     TEST_TEAM_NAME, TEST_JOB_NAME)))));
+    // Check authentication credentials are filled properly before continuing test.
+    Assertions.assertFalse(iamServiceAccountAccessKeyId.isBlank());
```

Comment:
What is the behavior of @Value("datajobs.aws.serviceAccountSecretAccessKey") if it is not present - null or empty?

Reply:
By default it should be empty.